clsCompositeCommon/CLSAcquamanCommon.pri

```diff
@@ -0,0 +1,93 @@
+
+DESTDIR = build
```

Comment:
I understand the desire to move the CLS common stuff into the same place.  But I don't like having the beamline specific pieces in a pri file that specifically designates CLS.  It's like finding an IDEAS class in the VESPERS pri file.  If we are going to split them up, then it needs to be more thorough.